### PR TITLE
Separately handle operator_tag and component_tag

### PR DESCRIPTION
**Pull request by tiraboschi**
Requires: https://github.com/kubevirt/web-ui-operator/pull/91
Fixes: https://github.com/kubevirt/hyperconverged-cluster-operator/issues/160

### CODE REVIEW

File: deploy/converged/olm-catalog/kubevirt-hyperconverged/0.0.1/kubevirt-hyperconverged-operator.v0.0.1.clusterserviceversion.yaml [hunk 1]
```diff
@@ -1372,7 +1372,7 @@ spec:
                   - name: OPERATOR_TAG
                     value: v0.1.10
                   - name: WEBUI_TAG
-                    value: v0.1.10
+                    value: v2.0.0-14.8
```
Comment (djzager) [inline deploy/converged/olm-catalog/kubevirt-hyperconverged/0.0.1/kubevirt-hyperconverged-operator.v0.0.1.clusterserviceversion.yaml]: This looks like a d/s value. Is it?

Comment (tiraboschi) [reply deploy/converged/olm-catalog/kubevirt-hyperconverged/0.0.1/kubevirt-hyperconverged-operator.v0.0.1.clusterserviceversion.yaml]: No, the whole point about this is that upstream web-ui releases are tagged as downstream while web-ui-operator is not.
See:
https://github.com/kubevirt/web-ui/releases
https://quay.io/repository/kubevirt/kubevirt-web-ui?tab=tags

Comment (djzager) [reply deploy/converged/olm-catalog/kubevirt-hyperconverged/0.0.1/kubevirt-hyperconverged-operator.v0.0.1.clusterserviceversion.yaml]: I follow now.

**Comment by rthallisey** [comment]
ci test please

**Comment by tiraboschi** [comment]
> ci test please

it will fail until https://github.com/kubevirt/web-ui-operator/pull/91 got merged and re-vendored

**Comment by tiraboschi** [comment]
https://github.com/kubevirt/web-ui-operator/pull/91 got merged but now we need a new tagged release of  web-ui-operator to re-vendor it here.

**Comment by tiraboschi** [comment]
ci test please

**Pull request merged**